lms/views/status.py

```diff
@@ -11,6 +11,6 @@ def status(request):
     try:
         request.db.execute("SELECT 1")
         return {"status": "okay"}
-    except Exception:  # pylint:disable=broad-except
```

Comment:
I'm not entirely sure what's going on with some of the apparently no-longer-needed disables. Isn't this still a broad exception? :man_shrugging: